src/gui/transferlistwidget.cpp

```diff
@@ -818,6 +820,22 @@ void TransferListWidget::applyToSelectedTorrents(const std::function<void (BitTo
     }
 }
 
+bool TransferListWidget::hasTorrentPreviewableFiles(const BitTorrent::TorrentHandle *const torrent) const
+{
+    if (!torrent->hasMetadata()) {
+        return false;
+    }
+
+    for (int i = 0; i < torrent->filesCount(); ++i) {
+        const QString ext = Utils::Fs::fileExtension(torrent->fileName(i));
```

Comment:
Redundant variable.

Reply:
Do you mean `ext`?

Reply:
Sure, there is no other ones in that line.

Reply:
That is because of readability / clarity, but I change it